data/json/items/ammo.json

```diff
@@ -1440,7 +1440,7 @@
         "name_plural" : "5.56 NATO SS109",
         "symbol" : "=",
         "color" : "yellow",
-        "description" : "5.56x45mm ammunition with a 62gr FMJ bullet containing a steel penetrator. It's light weight, high accuracy and managable recoil lead to it's use by NATO countries and many more. The round's yaw and fragmentation cause significant damage inside a target.",
```

Comment:
It's just me or the sentence "It's light weight, high accuracy and manageable recoil lead to it's use by NATO countries and many more." sounds weird, even with proper spelling? I don't want to rephrase it by myself (since I'm not a native speaker and I don't want to hurt Rivet or someone else :-_)_ but I'd be glad to implement better description if someone can propose it.